stock_available_unreserved/models/product.py

```diff
@@ -34,8 +34,11 @@ def action_open_quants_unreserved(self):
         quant_ids = quants.filtered(
             lambda x: x.product_id.qty_available_not_res > 0
         ).ids
-        result = self.env.ref("stock.product_template_open_quants").read()[0]
-        result["domain"] = [("id", "in", quant_ids)]
+        result = dict()
```

Comment:
This is not correct, if `product_template_open_quants` action does not exist anymore you need to use a new one or complete it here.